cartographer_grpc/sensor/serialization.h

```diff
@@ -28,25 +29,34 @@ namespace sensor {
 
 void CreateSensorMetadata(const std::string& sensor_id, int trajectory_id,
                           proto::SensorMetadata* proto);
+
```

Comment:
We only use newlines to group methods that are somehow belong to one another. So please remove the newlines.

Reply:
Done.